frontstage/common/api_call.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+import requests
+from requests.exceptions import ConnectionError, ConnectTimeout
+from structlog import wrap_logger
+
+from frontstage import app
+from frontstage.exceptions.exceptions import FailedRequest, InvalidRequestMethod
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def api_call(method, endpoint, parameters=None, json=None, headers=None):
```

Comment:
Is there a need to handle anything other than a 'get' method as it's currently the only usage?

Reply:
This method is going to handle all requests from frontstage in the near future, I added these in so it's ready for these in advance.